integration-test/trace-recorder/src/main/resources/integrationTestData/kotlinx.collections.immutable/tests.contract.list.ImmutableListTest_empty.txt

```diff
@@ -3,643 +3,643 @@ ImmutableListTest.empty() at :0
-                CompareContext@1.expected ➜ EmptyList@1 at ComparisonDSL.kt:21
-                CollectionBehaviorsKt$$Lambda@1.invoke(EmptyList@1): 1 at ComparisonDSL.kt:21
-                  EmptyList@1.hashCode(): 1 at CollectionBehaviors.kt:98
-                CompareContext@1.actual ➜ SmallPersistentVector@1 at ComparisonDSL.kt:21
-                CollectionBehaviorsKt$$Lambda@1.invoke(SmallPersistentVector@1): 1 at ComparisonDSL.kt:21
-                  SmallPersistentVector@1.hashCode(): 1 at CollectionBehaviors.kt:98
+              equalityBehavior.propertyEquals("hashCode", CollectionBehaviorsKt$$Lambda@1) at CollectionBehaviors.kt:98
+                expected ➜ EmptyList@1 at ComparisonDSL.kt:21
+                getter.invoke(EmptyList@1): 1 at ComparisonDSL.kt:21
+                  propertyEquals.hashCode(): 1 at CollectionBehaviors.kt:98
+                actual ➜ SmallPersistentVector@1 at ComparisonDSL.kt:21
+                getter.invoke(SmallPersistentVector@1): 1 at ComparisonDSL.kt:21
+                  propertyEquals.hashCode(): 1 at CollectionBehaviors.kt:98
                 AssertionsKt.assertEquals(1, 1, "hashCode") at ComparisonDSL.kt:21
               "".append(""): "" at CollectionBehaviors.kt:100
               "".append("toString"): "toString" at CollectionBehaviors.kt:100
               "toString".toString(): "toString" at CollectionBehaviors.kt:100
-              CompareContext@1.propertyEquals("toString", CollectionBehaviorsKt$$Lambda@1) at CollectionBehaviors.kt:100
-                CompareContext@1.expected ➜ EmptyList@1 at ComparisonDSL.kt:21
-                CollectionBehaviorsKt$$Lambda@1.invoke(EmptyList@1): "[]" at ComparisonDSL.kt:21
-                CompareContext@1.actual ➜ SmallPersistentVector@1 at ComparisonDSL.kt:21
-                CollectionBehaviorsKt$$Lambda@1.invoke(SmallPersistentVector@1): "[]" at ComparisonDSL.kt:21
+              equalityBehavior.propertyEquals("toString", CollectionBehaviorsKt$$Lambda@1) at CollectionBehaviors.kt:100
+                expected ➜ EmptyList@1 at ComparisonDSL.kt:21
+                getter.invoke(EmptyList@1): "[]" at ComparisonDSL.kt:21
+                actual ➜ SmallPersistentVector@1 at ComparisonDSL.kt:21
+                getter.invoke(SmallPersistentVector@1): "[]" at ComparisonDSL.kt:21
                 AssertionsKt.assertEquals("[]", "[]", "toString") at ComparisonDSL.kt:21
           CollectionBehaviorsKt.collectionBehavior$default(CompareContext@1, null, false, 3, null) at CollectionBehaviors.kt:10
             CollectionBehaviorsKt.collectionBehavior(CompareContext@1, "", true) at CollectionBehaviors.kt:104
               "".append(""): "" at CollectionBehaviors.kt:105
-              "".length(): 0 at CollectionBehaviors.kt:105
+              objectName.length(): 0 at CollectionBehaviors.kt:105
               "".append(""): "" at CollectionBehaviors.kt:105
```

Comment:
Should we use pretty printing for owner names? I would argue using them for function parameters only.